src/store/configs/index.ts

```diff
@@ -32,3 +33,35 @@ export const scanStoreApi = createApi({
   baseQuery: fetchBaseQuery({ baseUrl: '/scan-api' }),
   endpoints: () => ({}),
 });
+
+export const serviceNowStoreApi = createApi({
+  reducerPath: 'serviceNowApi',
+  baseQuery: fetchBaseQuery({
+    baseUrl:
+      'https://api-radix-servicenow-proxy-prod.radix.equinor.com/api/v1/',
```

Comment:
```suggestion
      baseUrl: configVariables.SERVICENOW_PROXY_BASEURL,
```
Use base url from config. Add the following to the import section:
import { configVariables } from '../../utils/config';

Reply:
fixed